lib/tests/misc.nix

```diff
@@ -678,6 +678,11 @@ runTests {
     ("%20%3F%26%3D%23%2B%25%21%3C%3E%23%22%7B%7D%7C%5C%5E%5B%5D%60%09%3A%2F%40%24%27%28%29%2A%2C%3B" == strings.escapeURL " ?&=#+%!<>#\"{}|\\^[]`\t:/@$'()*,;")
   ];
 
+  testToSentenceCase = {
+    expr = strings.toSentenceCase "hello world";
+    expected = "Hello world";
+  };
+
```

Comment:
You could add another test for when the value is a `path`
That behavior is kind of undefined if we don't add a test for it. With your curent implementation paths are automatically converted into strings which triggers a store import.

You might want to use `lib.isString` to prevent suprises.